Ollivanders/src/plugin.yml

```diff
@@ -1,6 +1,6 @@
 name: Ollivanders2
 main: net.pottercraft.ollivanders2.Ollivanders2
-version: 2.6.5
+version: 2.6.6
```

Comment:
2.6.5 is the correct "next" version. Current released version is 2.6.4.

Reply:
My bad, will replace this.